recipe/meta.yaml

```diff
@@ -1,15 +1,17 @@
-{% set version = "2.1.0" %}
+{% set version = "2.1.1" %}
 
 package:
   name: ocgis
   version: {{ version }}
 
 source:
-  url: https://github.com/NCPP/ocgis/archive/v{{ version }}.tar.gz
-  sha256: 9cd680ce2bf8a25f50c4123145f16806b63484bc1b5d10408acd1d144f828eaa
+  git_url: https://github.com/NCPP/ocgis.git
+  git_tag: master
```

Comment:
@bekozi we prefer tags or tarballs, the `master` branch is "fluid" and this build will be impossible to reproduce that way.

Reply:
@ocefpaf Yup, should have mentioned this was the initial test. There is usually a commit or two before the final release (maybe or maybe not). Is this frowned upon? Don't want to eat up the CI time unnecessarily.

Reply:
Nope. Testing the build is fine. Go for it!